bin/pca-import

```diff
@@ -105,7 +105,7 @@ def output_assessment_template():
     assessment["team_lead"] = "Test Testerton"
     assessment["start_time"] = "2017-01-01"
     assessment["end_time"] = "2017-01-21"
-    print(util.to_json(assessment))
+    print(util.to_json(assessment,sort_keys=False))
```

Comment:
Can you leave a comment here explaining why you're using `sort-keys=False` in this one place?  Otherwise this will be a great mystery when someone comes back to it in a year.

Reply:
See here a423b29